test/sanitizer_test.rb

```diff
@@ -271,7 +271,8 @@ def test_should_allow_custom_tags_with_custom_attributes
 
   def test_scrub_style_if_style_attribute_option_is_passed
     input = '<p style="color: #000; background-image: url(http://www.ragingplatypus.com/i/cam-full.jpg);"></p>'
-    assert_equal '<p style="color: #000;"></p>', safe_list_sanitize(input, attributes: %w(style))
+    actual = safe_list_sanitize(input, attributes: %w(style))
+    assert_includes(['<p style="color: #000;"></p>', '<p style="color:#000;"></p>'], actual)
```

Comment:
Sorry for not explaining better. The dependency on loofah is pretty loose, `spec.add_dependency "loofah", "~> 2.3"`, so I erred on the cautious side and made this test cover the pre-2.9 whitespace and the newer conventions.

We could also have just bumped the dependency on Loofah, but I'd defer to one of the Rails team to make that decision, because I don't have a good mental model for how people upgrade their Rails app dependencies.